frontend/src/components/AdminModals/AdminSuspendModal.js

```diff
@@ -27,22 +27,25 @@ function AdminSuspendModal(props) {
     </button>
   );
 
+  const toggleSuspendText = (isSuspended) => {
+    const textForSuspendedUser = 'Clicking "UnSuspend" will allow the user to logging into the app. The user will remain unsuspended until this status is removed by an administrator.If you do not wish to unsuspend the user, close this dialog box.';
+    const textForActiveUser = 'Clicking "Suspend" will prevent the user from logging into the app. The user will remain suspended until this status is removed by an administrator.If you do not wish to suspend the user, close this dialog box.';
```

Comment:
I'd break these out into multiline strings, too, as they're presently too wide. You can build them in chunks, like:

```
const textFoo = 'Blah blah blah ' +
  'blah blah blah' +
  'blah.'
```

That'll make them easier to read and obey our line width limits.